lib/language_pack/cache.rb

```diff
@@ -58,7 +58,12 @@ def load_without_overwrite(path, dest=nil)
     return unless @cache_base
 
     dest ||= path
-    copy (@cache_base + path), dest, '-a --update=none'
+
+    if ENV["STACK"] == "heroku-20"
```

Comment:
@schneems This needs to be both Heroku-20 and Heroku-22, since only Heroku-24 has the new flag. See the man page links I included in:
https://github.com/heroku/heroku-buildpack-ruby/pull/1583#discussion_r2063787708